CedarJava/src/main/java/com/cedarpolicy/serializer/JsonEUID.java

```diff
@@ -56,6 +57,7 @@ public JsonEUID(String type, String id) {
         this.type = type; this.id = id;
     }
 
+    @SuppressFBWarnings
```

Comment:
Nit: if there are specific new FindBugs warnings that we don't agree with and want to suppress, can we include the particular ones in the annotation? Just so we don't end up masking useful ones which might be introduced later.

Reply:
For reference, the warning is due to throwing exceptions in constructors:

```
M B CT: Exception thrown in class com.cedarpolicy.model.slice.Policy at new com.cedarpolicy.model.slice.Policy(String, String) will leave the constructor. The object under construction remains partially initialized and may be vulnerable to Finalizer attacks.  At Policy.java:[line 55]
```

There's a separate PR to update the findbugs dependency we've been holding off on merging until we've decided what we want to do.

Reply:
Thanks for feedback, I will push an update to list the specific rule for now.

Reply:
The other pull request is from dependabot and does not include the required constructor changes. If you prefer that I back out the SpotBugs upgrade and annotations, that is certainly an option.

Reply:
@lucajone and @john-h-kastner-aws I pushed a commit adding the specific SpotBugs rules to the suppress annotations.

If you would prefer that I back out the SpotBugs plugin upgrade from this PR, I can make adjustments.

Reply:
We're all happy with it.